src/main/java/com/spotinst/sdkjava/model/bl/admin/messageCenter/GetMessage.java

```diff
@@ -0,0 +1,375 @@
+package com.spotinst.sdkjava.model.bl.admin.messageCenter;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class GetMessage {
```

Comment:
If this a response object rename it to GetMessageResponse

Reply:
Resolved